src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -11,7 +12,7 @@ namespace System.Numerics
     /// are real numbers, and i is the imaginary unit, with the property i2= -1.
     /// </summary>
     [Serializable]
-    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    [TypeForwardedFrom("System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public readonly struct Complex : IEquatable<Complex>, IFormattable
```

Comment:
Since nothing is actually changing for Complex, I'd think it more ideal to not mix some whitespace/style changes in with this PR.

Reply:
I'd rather not spin CI again to remove the change here if unnecessary. Same with the `Math.E` change, I can easily get in the PR that handles `WriteLittleEndian` and friends and avoid running 165 jobs unnecessarily.

Its a minor cleanup in a type I own and am responsible for anyways and which I'd end up fixing when Complex is unblocked anyways.